### PR TITLE
Issue #2848 use minishift command to get status for systemtray

### LINKED ISSUE #2848: Start/Stop of profile from the tray does not work as expected in windows
Some time after the tray has started, trying to start or stop a profile from it in windows opens up cmd prompt showing the localhost ip address with some random port number.

https://github.com/minishift/minishift/wiki/Known-issues#windows
Ref: https://github.com/minishift/minishift/pull/2633#issuecomment-425339621

image

### PR DESCRIPTION
**Pull request by anjannath**
Fixes #2848

### CODE REVIEW

**Comment by amitkrout** [comment]
Sure

**Comment by amitkrout** [comment]
@anjannath I am not able to get the bit from circle ci so using the artifacts form https://ci.appveyor.com/project/minishift-bot/minishift/builds/19785944/artifacts

**Comment by anjannath** [comment]
@amitkrout The bits for [windows](https://5501-62728051-gh.circle-artifacts.com/0/ms/windows-amd64/systemtray/minishift.exe) and [darwin](https://5501-62728051-gh.circle-artifacts.com/0/ms/darwin-amd64/systemtray/minishift)

**Comment by anjannath** [comment]
> @anjannath I am not able to get the bit from circle ci so using the artifacts form https://ci.appveyor.com/project/minishift-bot/minishift/builds/19785944/artifacts

Only circle ci builds with systemtray, appveyor bits won't have it.

**Comment by amitkrout** [comment]
> Only circle ci builds with systemtray, appveyor bits won't have it.

Right, day before yesterday you told me. Verifying the bits anyway

**Comment by amitkrout** [comment]
Verify the Windows bit. Proper start/stop log is being flashed on the command prompt. Also expected ```minishift status``` is reflected in the terminal.

Fix works on Win10+Hyper-V

File: pkg/minishift/systemtray/systemtray.go [hunk 1]
```diff
@@ -106,21 +102,20 @@ func OnExit() {
...
+	out, _ := command.Output()
+	stdOut := fmt.Sprintf("%s", out)
 
-	vmStatus, err := cluster.GetHostStatus(api, profileName)
-	if err != nil {
-		return DOES_NOT_EXIST
-	}
-
-	if vmStatus == libmachineState.Running.String() {
+	if strings.Contains(stdOut, "Running") {
```
Comment (LalatenduMohanty) [inline pkg/minishift/systemtray/systemtray.go]: Now the code is more consistent with rest of the code IMO :-)

**Pull request merged**